utils.py

```diff
@@ -30,3 +30,58 @@ def center_of_tile(x: int, y: int) -> Vector:
         ((x // TILE_SIZE) * TILE_SIZE) + TILE_SIZE // 2,
         ((y // TILE_SIZE) * TILE_SIZE) + TILE_SIZE // 2,
     )
+
+
+def process_objects(file_path: str) -> list:
+    """
+    Reads the .tmx file that the tile infomation is stored in an process the object infomation into
+    a list of dicts
+    :return: list constaining a dictionary for each layer
+    """
+
+    guards = []
+
+    file = minidom.parse(file_path)
+    objects = file.getElementsByTagName("objectgroup")
+
+    for i in objects:
+        guard = {'name': i.getAttribute('name'),
+                 'objects': [],
+                 'object_count': 0}
+
+        child_object_elements = i.getElementsByTagName('object')
+
+        for x in child_object_elements:
+            guard['object_count'] += 1
+            guard['objects'].append({'name': x.getAttribute('name'),
```

Comment:
Try using black, this should clean this formatting maybe?
Install pre-commit -> `pip install pre-commit`
Install hooks -> `pre-commit install`
Run  -> `pre-commit run`